docs/rules/no-deprecated-api.md

```diff
@@ -151,12 +151,12 @@ For example of `package.json`:
     "name": "your-module",
     "version": "1.0.0",
     "engines": {
-        "node": ">=8.0.0"
+        "node": ">=16.0.0"
     }
 }
 ```
 
-If you omit the [engines] field, this rule chooses `>=8.0.0` as the configured Node.js version since `8` is the minimum version the community is maintaining (see also [Node.js Release Working Group](https://github.com/nodejs/Release#readme)).
+If you omit the [engines] field, this rule chooses `>=16.0.0` as the configured Node.js version since `16` is the active lts (see also [Node.js Release Working Group](https://github.com/nodejs/Release#readme)).
```

Comment:
It's 18 that's the active LTS, both 14 and 16 are simply maintained LTS versions.

We should be targeting the oldest maintained release, hence I suggest waiting until after April 30 to merge this, as April 30 will be when Node 14 is expected to exit LTS